cogs5e/funcs/sheetFuncs.py

```diff
@@ -175,8 +176,12 @@ def critSub(matchobj):
                     critDice = re.sub(r'(\d+)d(\d+)', critSub, damage_str)
             else:
                 critDice = damage_str
+            critDice = parsemax(critDice,maxdice)
```

Comment:
this is an unbound local error

Reply:
Also, why is this in parsecrit?

Reply:
Because you called parsecrit in like 3 different places and i wasn't sure where would be a good place to parse for max